apps/dashboard/app/helpers/batch_connect/sessions_helper.rb

```diff
@@ -186,40 +96,11 @@ def session_save_errors(errors)
   def connection_tabs(id, tabs)
```

Comment:
all the complexity of this is to render one partial if there is only one tab another partial if there are multiple tabs, presumably to hide the superfluous tab if there is only 1

Instead, we can just use multiple tabs partial, always display the tab if there is only one, and then use this css rule to hide the case there is only 1 tab:

    .session-panel .nav.nav-tabs li:only-child { display: none; }

But the tab does provide some information (type of connection it is). So maybe we just want to keep it and not use the css rule. We can discuss.